Alpaca.Markets/Enums/HistoryPeriod.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Alpaca.Markets
+{
+    /// <summary>
+    /// Period values for portfolio history in the Alpaca REST API.
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum HistoryPeriod
+    {
+        /// <summary>
+        /// 1 month
+        /// </summary>
+        [EnumMember(Value = "1M")]
+        Month,
+
+        /// <summary>
+        /// 3 month
+        /// </summary>
+        [EnumMember(Value = "3M")]
+        ThreeMonth,
```

Comment:
Wasn't sure about the naming convention we should be using here. Feel free to let me know if something else is preferred.